### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by Prosquadmama** [quickemu-project/quickemu#1137] [Quickemu] [answered]
I am using a fedora and i do have the new version but everytime im getting this error : [cyber@cyberbox qemu_mode]$ qemu-system-x86_64 --version
QEMU emulator version 7.2.10 (qemu-7.2.10-1.fc38)
Copyright (c) 2003-2022 Fabrice Bellard and the QEMU Project developers
[cyber@cyberbox qemu_mode]$ ./build_qemu_support.sh
=================================================
AFL binary-only instrumentation QEMU build script
=================================================

[*] Performing basic sanity checks...
[+] All checks passed!
[+] Cryptographic signature on  checks out.
[*] Uncompressing archive (this will take a while)...
tar: : Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
[cyber@cyberbox qemu_mode]$

**Comment by Prosquadmama** [comment]
can i pplease maybe get the code for ./build_qemu_support.sh and it might help ??

**Comment by lj3954** [comment, marked as answer]
The quickemu project has nothing to do with this build_qemu_support.sh script, which appears to have been created by Google. Also, Fedora 38 is a year old and is losing support later this month, so you should definitely upgrade.
